Makefile

```diff
@@ -27,6 +27,9 @@ OUTPUT_RTRMON := $(DIST_DIR)rtrmon$(SUFFIX)
 export CGO_ENABLED ?= 0
 
 
+.PHONY: build-all
+build-all: vet build-stayrtr build-rtrdump build-rtrmon
+
 .PHONY: vet
 vet:
 	go vet cmd/stayrtr/stayrtr.go
```

Comment:
Separate change,  but it might be more idiomatic to  also vet all the friends if the default is to build all the friends, ?

Reply:
Yeah I agree, was gonna get around to it later